src/js/components/Avatar/Avatar.js

```diff
@@ -49,12 +52,18 @@ const Avatar = ({
 
   if (typeof src === 'string') {
     return (
-      <StyledAvatar {...avatarProps} {...rest} background={`url(${src})`} />
+      <StyledAvatar {...avatarProps} {...rest}>
+        <Image a11yTitle={a11yTitle || ariaLabel} fit="contain" src={src} />
```

Comment:
It seems inconsistent to put `a11yTitle` in the child `Image` here but in the parent `StyledAvatar` below. I would think they should always be at the same place in the DOM.

Reply:
That's fair to be consistent it should be on the `StyledAvatar`

Reply:
I think when we have an Image needs an a11yTitle. Also it looks like we are getting an accessibility error for putting the aria-label on a div without a role.

<img width="644" alt="Screen Shot 2022-10-04 at 3 49 24 PM" src="https://user-images.githubusercontent.com/54560994/193936767-0ad36e2b-de45-45ad-b033-7a9063b1ef1e.png">

Reply:
Ill look